.gitignore

```diff
@@ -37,6 +37,8 @@ npm-debug.log
 testem.log
 /typings
 package-lock.json
+yarn.lock
+yarn-error.log
```

Comment:
These changes are not related to this PR, I think we should revert these changes.